lib/pbench/server/api/resources/query_apis/controllers_list.py

```diff
@@ -22,20 +22,22 @@ def __init__(self, config: PbenchServerConfig, logger: Logger):
             config,
             logger,
             Schema(
-                Parameter("user", ParamType.USER, required=False),
+                Parameter("user", ParamType.USER, required=False, nullable=True),
+                Parameter("access", ParamType.ACCESS, required=False),
```

Comment:
The allowance for `""` in nullable is specifically for the dashboard's current behavior; I've waffled on changing that. It would make more sense for `nullable` to allow only `None`, although it may be that we really don't even _need_ `nullable` at all for the #2370 semantics. It was originally there so `"user": None` would mean "public", but the addition of an `access` parameter complicates (or simplifies) the semantics, based on our subsequent discussions and #2370.

Really, I'd started out doing `nullable`, decided we probably don't need it for user/access, but felt bad dropping that because I didn't want to undo my work to implement `nullable` in the first place.

Reply:
To be honest, what I want is to get to the #2370 model...so, if we get there faster and better by not discussing this further, I'm at peace with that.  I just don't want to get caught out when we're straddling the gap....  😬